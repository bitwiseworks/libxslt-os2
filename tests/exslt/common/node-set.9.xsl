<?xml version="1.0"?>

<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="1.0"
xmlns:exslt="http://exslt.org/common"
extension-element-prefixes="exslt">

<!-- https://gitlab.gnome.org/GNOME/libxslt/-/issues/37 -->

<xsl:variable name="tree1">
  <a><b><c><d/></c></b></a>
</xsl:variable>
<xsl:variable name="tree2">
  <a><b><c><d/></c></b></a>
</xsl:variable>

<xsl:template match="a">
  <xsl:for-each select="/*">
    <match/>
  </xsl:for-each>
</xsl:template>

<xsl:template match="/">
  <out>
    <xsl:apply-templates select="exslt:node-set($tree1)/a | exslt:node-set($tree2)/a"/>
  </out>
</xsl:template>

</xsl:stylesheet>
