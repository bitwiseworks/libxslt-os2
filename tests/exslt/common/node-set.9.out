<?xml version="1.0"?>
<out><match/><match/></out>
